ghost/core/test/integration/services/email-service/email-event-storage.test.js

```diff
@@ -209,9 +209,6 @@ describe('EmailEventStorage', function () {
         const providerId = emailBatch.provider_id;
         const timestamp = new Date(2000, 0, 1);
 
-        const {body: {members: [member]}} = await agent.get(`/members/${memberId}`);
```

Comment:
I've removed the checks from this test in favor of moving them into their own test file below